include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/IDEInstInteractionAnalysis.h

```diff
@@ -253,26 +261,28 @@ class IDEInstInteractionAnalysisT
             // If a value is stored that holds we must generate all potential
             // memory locations the store might write to.
             if (Store->getValueOperand() == src || ValuePTS->count(src)) {
+              Facts.insert(Store->getValueOperand());
               Facts.insert(Store->getPointerOperand());
               Facts.insert(PointerPTS->begin(), PointerPTS->end());
             }
-            // If the value to be stored does not hold we must at least add
-            // the store instruction and the points-to set as the instruction
-            // still interacts with the memory locations pointed to be PTS.
-            if (Store->getPointerOperand() == src || PointerPTS->count(src)) {
-              Facts.insert(Store);
-              Facts.erase(src);
-            }
+            // If the value to be stored does not hold we must at least add the
+            // store instruction and the points-to set as the instruction still
+            // interacts with the memory locations pointed to be PTS.
+            // if (Store->getPointerOperand() == src ||
+            //        PointerPTS->count(src)) {
+            //   Facts.insert(Store);
+            //  Facts.erase(src);
+            // }
```

Comment:
Why is this commented? Not needed or did you miss replacing it?

Reply:
I need to double check on that one. There was some issue here ...